libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus/hwdef.dat

```diff
@@ -22,14 +22,60 @@ APJ_BOARD_ID 1063
 
 undef CHECK_ICM20602_EXT
 undef CHECK_IMU0_PRESENT
+undef CHECK_IMU1_PRESENT
+undef CHECK_IMU2_PRESENT
+undef CHECK_ICM20649
+undef BOARD_VALIDATE
 undef IMU
+undef I2C_ORDER
 
+PF14 I2C4_SCL I2C4
+PF15 I2C4_SDA I2C4
+
+define HAL_SKIP_AUTO_INTERNAL_I2C_PROBE
+define HAL_I2C_INTERNAL_MASK 4
+
+I2C_ORDER I2C2 I2C1 I2C4
```

Comment:
does this change compass probe order? I think it will if user has a compass on each bus I think they will change order

Reply:
@tridge It changes the probe order but bus id remains same. Since we have PRIO_ID stuff even if the probe order is different, the compass order and calibration values will be retained from old setup. Only in the case of clean slate will the order be different.